webrtc.html

```diff
@@ -5396,10 +5396,87 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
     </table>
   </section>
 
-  <section>
+  <section class="informative">
     <h2>Security Considerations</h2>
+    <p>This section is non-normative; it specifies no new behaviour,
+    but instead summarizes information already present in other parts
+    of the specification.</p>
+
+    <p>This document extends the Web platform with the ability to set
+    up real time, direct communication between browsers and other
+    devices, including other browsers.</p>
+
+    <p>This means that data and media can be shared between
+    applications running in different browsers, or between an
+    application running in the same browser and something that is not a
+    browser, something that is an extension to the usual barriers in
+    the Web model against sending data between entities with different
+    origins.</p>
+
+    <p>The WebRTC specification provides no user prompts or chrome
+    indicators for communication; it assumes that once the Web page
+    has been allowed to access media, it is free to share that media
+    with other entities as it chooses.</p>
+
+    <p>A mechanism, <code><a>peerIdentity</a></code>, is provided that gives
+    Javascript the option of requesting media that the same javascript
+    cannot access, but can only be sent to certain other entities.</p>
+
+    <p>Even without WebRTC, the Web server providing a Web application
+    will know the public IP
+    address to which the application is delivered. Setting up
+    communications exposes additional information about the browser’s
+    network context to the web application, including the set of
+    (possibly private) IP addresses available to the browser for WebRTC
+    use. Some of this information has to be passed to the
+    corresponding party to enable the establishment of a communication
+    session.</p>
+
+    <p>Revealing IP addresses can leak location and means of
+    connection; this can be sensitive.</p>
```

Comment:
Replying to 3 points in bullets:
- The fact that the connection to the webserver leaked an IP address is already in there.
- The Network API https://dvcs.w3.org/hg/dap/raw-file/tip/network-api/Overview.html has been discontinued. Last I looked at it, it didn't do anything with addresses; it was just "3G up, Ethernet down".
- In the Chrome investigation, we found not only that VPNs were troublesome, they were more diverse than a division into "split VPN" and "non split VPN" would account for. In particular, one particular type of leakage showed up only with one specific VPN product, not with other products designed to achieve the same purpose. This is too complex for a security considerations section.